### PR TITLE
:bug: Fix display type of component library not persistent

### PR DESCRIPTION
**Pull request by pabloalba**
Fixes https://tree.taiga.io/project/penpot/issue/4512

### CODE REVIEW

File: frontend/src/app/main/data/workspace.cljs [hunk 1]
```diff
@@ -1842,6 +1842,26 @@
...
+;; File Library persistent settings
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+
+(defn set-file-library-listing-thumbs?
+  [listing-thumbs?]
+  (ptk/reify ::set-file-library-listing-thumbs?
+    ptk/UpdateEvent
+    (update [_ state]
+      (assoc-in state [:workspace-global :file-library-listing-thumbs?] listing-thumbs?))))
```
Comment (niwinz) [inline frontend/src/app/main/data/workspace.cljs]: I think the function name and the state keyword should not end with `?`
Especially when the function is not a predicate.

File: frontend/src/app/main/data/workspace.cljs [hunk 2]
```diff
@@ -1842,6 +1842,26 @@
...
+
+(defn set-file-library-reverse-sort?
+  [reverse-sort?]
+  (ptk/reify ::set-file-library-reverse-sort?
+    ptk/UpdateEvent
+    (update [_ state]
+      (assoc-in state [:workspace-global :file-library-reverse-sort?] reverse-sort?))))
```
Comment (niwinz) [inline frontend/src/app/main/data/workspace.cljs]: The same here

File: frontend/src/app/main/refs.cljs [hunk 3]
```diff
@@ -183,6 +183,12 @@
 (def context-menu
   (l/derived :context-menu workspace-local))
 
+(def file-library-listing-thumbs?
+  (l/derived :file-library-listing-thumbs? workspace-global))
```
Comment (niwinz) [inline frontend/src/app/main/refs.cljs]: Here, the var name it is ok to be with `?`, but the state access should be changed accordingly to the previous change

File: frontend/src/app/main/data/workspace.cljs (hunk 1 above)
Comment (pabloalba) [reply frontend/src/app/main/data/workspace.cljs]: Fixed

File: frontend/src/app/main/data/workspace.cljs (hunk 2 above)
Comment (pabloalba) [reply frontend/src/app/main/data/workspace.cljs]: Fixed

File: frontend/src/app/main/refs.cljs (hunk 3 above)
Comment (pabloalba) [reply frontend/src/app/main/refs.cljs]: Fixed

**Pull request merged**